scala_proto/scala_proto.bzl

```diff
@@ -20,17 +20,8 @@ load(
 )
 
 def register_default_proto_dependencies():
-    if native.existing_rule("io_bazel_rules_scala/dependency/proto/grpc_deps") == None:
-        native.bind(
-            name = "io_bazel_rules_scala/dependency/proto/grpc_deps",
-            actual = "@io_bazel_rules_scala//scala_proto:default_scalapb_grpc_dependencies",
-        )
-
-    if native.existing_rule("io_bazel_rules_scala/dependency/proto/implicit_compile_deps") == None:
-        native.bind(
-            name = "io_bazel_rules_scala/dependency/proto/implicit_compile_deps",
-            actual = "@io_bazel_rules_scala//scala_proto:default_scalapb_compile_dependencies",
-        )
+    # for backwards compatibility register toolchain for deps
```

Comment:
I don't understand this.
Will this be forever? Is this temporary?

Reply:
Let's wait with this discussion when we have separate PR for proto deps toolchain